api/proto/metrics/metrics.proto

```diff
@@ -228,6 +228,9 @@ message logMetric {
 
   uint64 numDeviceBundleProtoBytesSent = 17;
   uint64 numAppBundleProtoBytesSent = 18;
+
+  uint64 avgDeviceBundleSendTimeMillis = 19; // protolint:disable:this FIELD_NAMES_LOWER_SNAKE_CASE
+  uint64 avgAppBundleSendTimeMillis = 20; // protolint:disable:this FIELD_NAMES_LOWER_SNAKE_CASE
```

Comment:
These protolints didn't silence prototool.

But you can define them in lower snake case and the generated fields will be CamelCase.

Reply:
Ok. I'll change these two members to lower snake case.